src/Tizen.UIKit/src/UIKit.Input/Interface.cs

```diff
@@ -0,0 +1,1182 @@
+#pragma warning disable CS1591
+using System;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.ComponentModel;
+namespace UIKit {
+
+namespace Input {
+
+/// <summary>An object implementing this interface can send pointer events.
+/// Windows and canvas objects may send input events.
+/// 
+/// A &quot;pointer&quot; refers to the main pointing device, which could be a mouse, trackpad, finger, pen, etc... In other words, the finger id in any pointer event will always be 0.
+/// 
+/// A &quot;finger&quot; refers to a single point of input, usually in an absolute coordinates input device, and that can support more than one input position at at time (think multi-touch screens). The first finger (id 0) is sent along with a pointer event, so be careful to not handle those events twice. Note that if the input device can support &quot;hovering&quot;, it is entirely possible to receive move events without down coming first.
+/// 
+/// A &quot;key&quot; is a key press from a keyboard or equivalent type of input device. Long, repeated, key presses will always happen like this: down...up,down...up,down...up (not down...up or down...down...down...up).</summary>
+/// <since_tizen> 6 </since_tizen>
+[UIKit.Input.InterfaceConcrete.NativeMethods]
+[UIKit.Wrapper.BindingEntity]
+public interface IInterface : 
```

Comment:
Please don't name your interface IInterface, because this name doesn't say what to expect from it to the user.